testnet/stacks-node/src/burnchains/mocknet_controller.rs

```diff
@@ -89,16 +90,22 @@ impl BurnchainController for MocknetController {
         &mut self,
         _ignored_target_height_opt: Option<u64>,
     ) -> Result<(BurnchainTip, u64), BurnchainControllerError> {
+        // If `config` sets a value, use that. Otherwise, use a default.
```

Comment:
Why is the epoch even configurable in the first place?

Reply:
It's very useful for testing to make the epochs configurable (see the epoch switch tests in the other PR), as well as for things like private testnet testing -- having exactly one vector of epochs for testnet and regtest is pretty painful.